translator/totomlconfig/tomlConfigs/tomlConfig.go

```diff
@@ -0,0 +1,338 @@
+package tomlConfigs
+
+type (
+	TomlConfig struct {
```

Comment:
What led to this struct? This feels weird to me because this is just a Go struct that mirrors the `schema.json`. Is there a way we can have one or the other, but not both? I understand that this is intended for the unit tests, but it strikes me as redundant.

Reply:
Good question, it is required by the BurntSushi toml parser, it's the object representing the whole config object, like Java you can simple parse a config into a POJO, this is the Go equivalent of that POJO.

Reply:
And we recently discover that schema.json is not a necessary thing to maintain, unfortunately this struct is, for the parser to work as intended.